graftm/clusterer.py

```diff
@@ -47,7 +62,8 @@ def uncluster_annotations(self, input_annotations, reverse_pipe):
                 placed_alignment_base = placed_alignment_file.replace('_clustered.fa', '')
             output_annotations[placed_alignment_base] = {}
             for rep_read_name, rep_read_taxonomy in cluster_classifications.iteritems():
-
+                if merge_reads:
+                    rep_read_name=rep_read_name[:-2]
```

Comment:
Well I thought I did originally, but it fails when normal reads are provided so obviously not. I'm rerunning them now.

________________________________
From: Ben J Woodcroft <notifications@github.com>
Sent: Tuesday, 26 April 2016 3:33:22 PM
To: geronimp/graftM
Cc: Mr Joel Boyd; Author
Subject: Re: [geronimp/graftM] geronimp-slash_error patch (#171)


In graftm/clusterer.py<https://github.com/geronimp/graftM/pull/171#discussion_r61030713>:

> @@ -47,7 +62,8 @@ def uncluster_annotations(self, input_annotations, reverse_pipe):
>                  placed_alignment_base = placed_alignment_file.replace('_clustered.fa', '')
>              output_annotations[placed_alignment_base] = {}
>              for rep_read_name, rep_read_taxonomy in cluster_classifications.iteritems():
> -
> +                if merge_reads:
> +                    rep_read_name=rep_read_name[:-2]


You've run unit tests on this branch yeh?

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/geronimp/graftM/pull/171/files/95d1aeed6efe520c3664fb5c79ef12db87c682e6#r61030713>